lua/swenv/api.lua

```diff
@@ -28,6 +28,55 @@ local set_venv = function(venv)
   end
 end
 
+---
+---Checks who appears first in PATH. Returns `true` if `first` appears first and `false` otherwise
+---
+---@param first string|nil
+---@param second string|nil
+---@return boolean
+local hasHighPriorityInPath = function (first, second)
```

Comment:
[nit] since everything else is snake_case could this also be?

Reply:
sure, force of habit I guess.